util/collections.go

```diff
@@ -49,3 +54,12 @@ func removeDuplicatesFromList(list []string, keepLast bool) []string {
 	}
 	return out
 }
+
+// Returns an HCL compliant formated list of strings
+func ListToHCLArray(list []string) string {
```

Comment:
This function makes it sound like it'll return a string of the format `["foo", "bar", "baz"]`, but in reality, it returns `"foo", "bar", "baz"` (no brackets). Therefore, it should probably be called something like `ToCommaSeparatedStrings` or similar.